src/nebula-hooks/use-virtualized-table.ts

```diff
@@ -53,9 +53,8 @@ const useVirtualizedTable = ({
   isFontLoaded,
   viewService,
 }: UseVirtualizedTable) => {
-  const isPrintingMode = isPrinting(layout, viewService);
-  const shouldRender = layout.usePagination === false && !isPrintingMode;
-  const tableData = useMemo(() => getVirtualScrollTableData(layout, interactions), [layout, interactions]);
+  const shouldRender = !renderAsPagination(layout, viewService);
```

Comment:
Centralizing this makes sure you render one and one only pagination or virtualized table 👍